db/migrate/20230123192830_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
```

Comment:
- [ ] You need to create `AuthorId` which is the foreign key coming from the `Author` table which is an alias to the `User` table. I think you should instead have:
- `:comments, :author, foreign_key: { to_table: 'users' }`

Reply:
well resolved! ✔️